### PR TITLE
Add notice about stopping cron jobs enablement

### PR DESCRIPTION
**Pull request by cotsog**

### CODE REVIEW

File: user/cron-jobs.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ layout: en
 permalink: /user/cron-jobs/
 ---
 
+> Update 2016-08-02: We will not enable cron jobs for new organizations/repositories for the time being and we unfortunately don't have an ETA regarding when we will restart enabling them. Sorry for the inconvenience.
```
Comment (BanzaiMan) [inline user/cron-jobs.md]: Needs to stand out more. With quoting, the text might be too light to draw attention.

**Comment by cotsog** [comment]
@plaindocs: do we have some CSS styling that could make the notice standout more than a quote e.g. some kind of warning box? Thanks!

**Comment by MariadeAnton** [comment]
@plaindocs hold this for a bit, please! Cron jobs should be back up now :)

**Comment by cotsog** [comment]
Cron jobs are fixed! \o/

**Pull request closed without merging**